Resources/Maps/_Mono/Shuttles/Nfsd/anura.yml

```
@@ -1402,148 +1415,152 @@
       parent: 1
 - proto: CircularShieldBase
   entities:
-  - uid: 474
+  - uid: 157
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)